tests/Unit/Evolution/Test_TagsDomain.cpp

```diff
@@ -115,12 +115,14 @@ void test() noexcept {
       domain::Tags::InertialMeshVelocityCompute<Dim>,
       evolution::domain::Tags::DivMeshVelocityCompute<Dim>>;
 
-  MAKE_GENERATOR(gen);
   const std::array<double, 3> velocity{{1.2, 0.2, -8.9}};
   const double initial_time = 0.0;
+  // in 1d TranslationY and TranslationZ are not used by the maps, and in 2d
+  // TranslationZ is not used by the maps.
```

Comment:
Good start, but maybe this is clearer? 
"In 1d, the helper function create_coord_map will only use the first name, i.e., TranslationX. In 2d, it uses the first two names, TranslationX and TranslationY."